{{ cookiecutter.app_name }}/apt.txt

```diff
@@ -0,0 +1,4 @@
+libpq-dev
+python-pip
+python-dev
+python3-dev
```

Comment:
py33 is still tested in travis, which runs tox, so I prefer to keep it configured.